util/arrayHelper.js

```diff
@@ -0,0 +1,75 @@
+module.exports.removeWhitespaceElements = function (arr) {
+    for(let i = 0; i < arr.length; i++) {
+        if(arr[i] === '') {
+            arr.splice(i, 1);
+            i--;
+        }
+    }
+    return arr;
+};
+
+// intended for intersections only
```

Comment:
A lot of this code looks duplicated. If its only used in one place it would be fine to remove from this file and keep in the code higher in this PR.

Reply:
@orangejulius That was my misstep. Pure duplicate of helper/intersectionsParsing.js. Deleted